packages/dx-react-grid/src/plugins/grouping-state.jsx

```diff
@@ -26,6 +27,16 @@ const adjustSortIndex = (sortIndex, grouping, sorting) =>
     0,
   );
 
+const getColumnGroupingEnabled = (columnExtensions, groupingEnabled) => (columnName) => {
```

Comment:
Let's move this to grid core. It may also be generic with the getColumnSortingEnabled.

Or maybe it will be more convenient to add thread parameter to the getColumnExtension extension that holds default value.

Reply:
Yes, it will be refactored here or in the upcoming related PR's. Can't choose a correct name)